lighthouserc.json

```diff
@@ -0,0 +1,34 @@
+{
+  "ci": {
+    "upload": {
+      "target": "temporary-public-storage"
+    },
```

Comment:
not sure if this is might be of [concern](https://github.com/GoogleChrome/lighthouse-ci/blob/main/docs/services-disclaimer.md#temporary-public-storage), or is there anyway to make use of the assertions without uploading reports?

Reply:
Regarding this, u can have a look at this: https://github.com/GoogleChrome/lighthouse-ci/blob/main/docs/configuration.md#target

I used the quick and easy solution, that will delete the report in a few days.

So, if you want, i would suggest to go the 'lhci' option, where we will upload the report into the lighthouse ci server, but need to create an account and need to make access to anyone who wants the report.

You can take a look at this for the 'lhci' option.
https://github.com/GoogleChrome/lighthouse-ci/blob/main/docs/server.md

Reply:
ok i just updated to make it save the reports as an artifact. 
Its a bit troublesome thou, need download it, unzip and open it in chrome to view the report.

Reply:
thats fine